src/main/java/com/stripe/model/SubscriptionScheduleInvoiceSettings.java

```diff
@@ -0,0 +1,21 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.model;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class SubscriptionScheduleInvoiceSettings extends StripeObject {
```

Comment:
Can you clarify why that one is top level instead of in `SubscriptionSchedule` and then used as `SubscriptionSchedule.InvoiceSettings` in the revision class?

Reply:
Now nested in `SubscriptionSchedule`